recipes/glaes/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This file is upstream no need to copy it here.

Reply:
Thanks for the suggestion. I was not aware i can reference it from there